snapshot/git/gitdb/db_test.go

```diff
@@ -228,27 +261,79 @@ func setup() (*dbFixture, error) {
 		return nil, err
 	}
 
+	tags, err := createRepo(tmp, "tags-repo")
```

Comment:
Fixed (though in a slightly different way)

You really don't trust the tmp reaper, huh? I expect that a Unix system will clean its tmp dir...